chart/compass/charts/director/config.yaml

```diff
@@ -30,6 +30,7 @@ graphql:
     formationTemplates: [ "formation_template:read" ]
     systemAuth: ["ory_internal"]
     systemAuthByToken: ["ory_internal"]
+    formationConstraint: [ "formation_constraint:read" ]
```

Comment:
Add `formationConstraint: [ "formation_constraint:read" ]` and `updateFormationConstraint: [ "formation_constraint:write" ]` in `/components/director/hack/config-local.yaml` as well.